ext/settings.html

```diff
@@ -841,7 +773,78 @@ <h1>Yomitan Settings</h1>
                 <label class="toggle"><input type="checkbox" data-setting="general.debugInfo"><span class="toggle-body"><span class="toggle-track"></span><span class="toggle-knob"></span></span></label>
             </div>
         </div></div>
-        <div class="settings-item advanced-only"><div class="settings-item-inner">
+        <div class="settings-item">
+            <div class="settings-item-inner settings-item-inner-wrappable">
+                <div class="settings-item-left">
+                    <div class="settings-item-label">Result grouping mode</div>
```

Comment:
I was considering the anki card creation aspect of it as impacting the appearance of the anki cards so it can be in appearance. But then again none of the other appearance options impact this... Feels a bit odd to have it in general though.

Reply:
![image](https://github.com/themoeway/yomitan/assets/24891609/29132983-b089-4442-855f-eb30d8d5c09c)
Yeah, seems like these three settings were intended to be grouped together as they deal with sorting and display of results. Not sure if they are worthy of their own section, 'general' is too abstract, 'popup appearance' matches only partially...

Reply:
I think having sorting dict under dict settings is really nice. The other two not sure... Neither of them feel important enough to be in general.

Reply:
I think "Result Display" or "Display" should be its own section separate from "Appearance".

In terms of areas of functionality:
"Result Display": What information to display in each entry
"Appearance": How to display each entry